demo-projects/meetup/site/lib/initApollo.js

```diff
@@ -19,8 +19,7 @@ function create(initialState, req) {
     link: createUploadLink({
       uri: `${serverUrl}/admin/api`, // Server URL (must be absolute)
       credentials: 'same-origin', // Additional fetch() options like `credentials` or `headers`
-      // Use fetch() polyfill on the server
-      fetch: !process.browser && fetch,
+      fetch,
```

Comment:
Made this use `fetch` unconditionally since that's what the Blog demo did. I'm not sure why it was made conditional, but it was causing an error in-browser. See https://github.com/jaydenseric/apollo-upload-client/issues/167 I figured it couldn't hurt to match what the Blog does.